cypress/integration/add-searched.ts

```diff
@@ -0,0 +1,36 @@
+export const addSearched = () => {
+  describe("Let's add some cards", () => {
+    it('Visits the Pokemon Collection Search Page', () => {
```

Comment:
unnecessary tests. For E2E tests, I would just check all the "Happy Path" interactions the user can do. In your case, the only tests you need is creating and deleting cards. You can also tests some stuff along the way too (search box, etc)